source/features/command-palette-vim-keybindings.tsx

```diff
@@ -0,0 +1,30 @@
+import select from 'select-dom';
+import onetime from 'onetime';
+
+import features from '.';
+
+function commandPaletteKeydown({key, ctrlKey}: KeyboardEvent): void {
+	if (!ctrlKey || (key !== 'n' && key !== 'p')) {
+		return;
+	}
+
+	const commandPalette = select('command-palette');
+
+	if (commandPalette) {
+		const targetKey = key === 'n' ? 'ArrowDown' : 'ArrowUp';
+		commandPalette.dispatchEvent(new KeyboardEvent('keydown', {bubbles: true, key: targetKey, code: targetKey}));
+	}
+}
+
+function init(): void {
+	document.addEventListener('keydown', commandPaletteKeydown);
+}
+
+void features.add(import.meta.url, {
+	shortcuts: {
+		n: 'Select next item in command palette',
+		p: 'Select previous item in command palette',
```

Comment:
```suggestion
		'ctrl n': 'Select next item in command palette',
		'ctrl p': 'Select previous item in command palette',
```

Reply:
I actually tried this first and what wound up happening was whenever I hit `ctrl` the next item would be automatically selected (rather than having to press `ctrl-n`). After holding `ctrl`, `n` and `p` would work, but I didn't want `ctrl` to automatically select the next item.

Reply:
Here is a video showing what I am talking about:

https://user-images.githubusercontent.com/174864/155544311-f4b69e63-9304-41c5-a7cc-7b9f68afdc19.mov

That is with this patch applied, and the only key I am pressing is `ctrl`:

```diff
diff --git a/source/features/command-palette-navigation-shortcuts.tsx b/source/features/command-palette-navigation-shortcuts.tsx
index cd81891e..bb824673 100644
--- a/source/features/command-palette-navigation-shortcuts.tsx
+++ b/source/features/command-palette-navigation-shortcuts.tsx
@@ -3,11 +3,7 @@ import onetime from 'onetime';
 
 import features from '.';
 
-function commandPaletteKeydown({key, ctrlKey}: KeyboardEvent): void {
-	if (!ctrlKey || (key !== 'n' && key !== 'p')) {
-		return;
-	}
-
+function commandPaletteKeydown({key}: KeyboardEvent): void {
 	const commandPalette = select('command-palette');
 
 	if (!commandPalette) {
@@ -24,8 +20,8 @@ function init(): void {
 
 void features.add(import.meta.url, {
 	shortcuts: {
-		n: 'Select next item in command palette',
-		p: 'Select previous item in command palette',
+		'ctrl n': 'Select next item in command palette',
+		'ctrl p': 'Select previous item in command palette',
 	},
 	awaitDomReady: false,
 	init: onetime(init),
```

Reply:
This only affects the text added to the shortcut help window (opened with `?`), not the actual implementation.

Reply:
Oh! I really should have seen that, I had to call `document.addEventListener` in `init` 🤦🏻.